### ISSUE TITLE
Adding a string

### ISSUE THREAD

**Issue by ImmersiveX** [NoCheatPlus/Issues#86]
Adding a string to the string section results in errors when hitting a mob or player.

String I am trying to add:
http://pastebin.com/UP4Brbrs
Error:
http://pastebin.com/i5DsuFwy

**Comment by asofold** [comment]
Could you post the full ouptut of the 'ncp version' command (text preferred over screen shots)? If it's an older version of NCP, i'd also like to know the ProtocolLib version in use, if in use and if not included  in the 'ncp version' info.

It looks like an older version of Spigot (1649), with an older version of NCP in use - also the ProtocolLib version might be outdated (should be 3.6.4 by now?).

**Comment by ImmersiveX** [comment]
The text is formatted upside down due to way that multicraft console works.


http://pastebin.com/27aMaV9j

**Comment by ImmersiveX** [comment]
Okay I figured out a fix for the newest build of NCP.
The actual problem is in ProtocolLib v3.6.4
After updating to ProtocolLib v3.6.5 (not yet released but the build #222 is the actual release) it has fixed the issue so far.
Thanks for looking into this but I eventually tried out all sorts of previous versions of the two plugins and figured out that this works.
NCP b925 & ProtocolLib v3.6.5 (build #222 heres the link: http://ci.dmulloy2.net/job/ProtocolLib/222/)

Thanks again, regards.
